### PR TITLE
#4267 Search bar network changes

### LINKED ISSUE #4267: network under the search bar
## Design
- It should not indicate what network you are on. 
- It has the same purpose as collection hover. 
- Look should be a bit different: 
image
- networks in a grey text are "soon" not clickable. 
- Active networks, those that you can click on are bold with different hover effect. Its in the landing page handoff. 
  - that is: 
image
## now
- now it indicates network you are on. Redirect is correct, just please change the visual so it isnt confusing.
image

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Context

- [x] Closes #4267
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [ ] My contribution builds **clean without any errors or warnings**
- [ ] I've merged recent default branch -- **main** and I've no conflicts
- [ ] I've tried to respect high code quality standards
- [ ] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
Screen Shot 2022-11-26 at 4 26 14 PM

### CODE REVIEW

**Comment by exezbcz** [comment]
Hi! 
Few things:
### Text format
- stroke should be outside, not inside. 
image
- if that is problem, maybe we can use k-grey on hover instead of the outline effect? 
- Can we please edit the networks - if there is beta, it would be in upper index and so on? Then it would maybe fit under the search bar and would not go beyond it. 

### Explore hover dropdown
image
- Then there's the explore hover dropdown; please fix this issue, which could be confusing.
- remove the bold text
- its not a indicator of network you are on

**Comment by prachi00** [comment]
I am not sure how to show `beta` at the top, since it seems to be there in the chain list as a whole text. @roiLeo any idea?

File: components/landing/SearchLanding.vue [hunk 1]
```diff
@@ -22,8 +22,8 @@
           v-for="chain in chainList"
           :key="chain.value"
           :class="[
-            'mr-2 mb-2',
-            'chain-option',
+            'mr-4 mb-2',
+            'chain-option active',
```
Comment (roiLeo) [inline components/landing/SearchLanding.vue:26]: everything will be `active` and `active active` if selected?
plus I find it more practical to change `mr-4 mb-2` with `m-2`

**Review by roiLeo — COMMENTED** [review]
> * stroke should be outside, not inside.

@prachi00 you could see implementation on "Kusama NFTS" text on landing

> I am not sure how to show `beta` at the top, since it seems to be there in the chain list as a whole text.

You could try to do something like:
```js
    if(chain.includes('[Beta]')) {
      chain.split(' ')[0]
    } 
```
and use `<sup>` html tag for beta text

**Review by roiLeo — COMMENTED** [review]
When hover on dark-mode, Beta text is difficult to read

Screenshot 2022-11-30 at 08-48-02 KodaDot - Kusama NFT Market Explorer

**Comment by exezbcz** [comment]
Hi! Im so sorry for changing it , but i dont think it looks good with the stroke and multiple networks. 
As a result, I'll keep with the basic hover color change, but this time we'll make the hover color darker. I used a fresh shade of grey.

- So if we can keep the text under search bar bold - text size is correct
- change the hover effect to simple color change
- new grey: hex #6B6B6B
image
image
its in landing page handoff - first page - colors https://www.figma.com/file/3iOjW12ERFRDSVnpLuhQmf/landing-page-handoff?node-id=1305%3A5612&t=77vynFQGSPCXNQAc-1

thanks

File: components/landing/SearchLanding.vue [hunk 2]
```diff
@@ -17,17 +17,17 @@
...
+        class="is-flex is-justify-content-center is-flex-wrap-wrap is-align-items-baseline">
         <a
           v-for="chain in chainList"
           :key="chain.value"
-          :class="[
-            'mr-2 mb-2',
-            'chain-option',
-            { active: urlPrefix === chain.value },
-          ]"
+          :class="['m-2', 'chain-option active']"
```
[the remark this answers is not in the document]
Comment (roiLeo) [reply components/landing/SearchLanding.vue:25]: `active` class is still present by default

**Comment by exezbcz** [comment]
Hi! few minor changes
- please make the "beta" normal text, not bold
image
- in dark mode the hover color is incorrect - should be k-shade. 
image

**Comment by exezbcz** [comment]
@prachi00 
> please make the "beta" normal text, not bold

File: components/landing/SearchLanding.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (roiLeo) [reply components/landing/SearchLanding.vue:26]: @prachi00 could you please remove active on all chain?

**Comment by yangwao** [comment]
it's missing some stuff but I want to see merged

pay 70 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $70 @ 30.88 USD/KSM ~ 2.267 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0xb2c6ef17816eaf66259dea71dae6908e2d0e432fb6a772d4158c166e0abcecd0](https://kusama.subscan.io/extrinsic/0xb2c6ef17816eaf66259dea71dae6908e2d0e432fb6a772d4158c166e0abcecd0)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**